base_station/gui/src/components/SATask.vue

```diff
@@ -308,7 +315,7 @@ export default {
                              "spectral chlorophyll drives"
                              "SAArm striptest drives"
                              "SAArm amino drives"
-                             "SAArm amino drives";
+                             "PDBFuse amino drives";
```

Comment:
does this get displayed? can it just be "PDB fuse amino drives"?

Reply:
PDBFuse is the name of the component, so if there's a space it won't display correctly.

Reply:
gotcha